pkg/fanal/analyzer/secret/secret.go

```diff
@@ -165,6 +166,9 @@ func (a *SecretAnalyzer) Required(filePath string, fi os.FileInfo) bool {
 		return false
 	}
 
+	if size := fi.Size(); size > 209_715_200 { // 200MB
```

Comment:
I'm not sure about the size
@knqyf263 wdyt?

Reply:
I feel even 10 MB is large for plain text.

Reply:
yes, I probably overdid it when I chose 200 MB :smile: 
updated in https://github.com/aquasecurity/trivy/pull/7085/commits/c3d5e4c60158fe0d8a152af467c90530cc3e5c2e

Reply:
Can we have this configurable i.e if value is passed through flag than we can consider it and skip the scanning?
If not provided then we can show the warning and proceed with the scanning.

```
--file-size-threshold 10mb
```

Reply:
I'm not sure new flag is good idea, because we will only be using it for `secret` scanner.
But we can think about adding this option to [secret config file](https://aquasecurity.github.io/trivy/v0.53/docs/scanner/secret/#configuration)

Reply:
Sure, anything would be fine, since this would be helpful a lot.
We can easily add a config and scan whole registry and based on the threshol, trivy will skip the scanning.

Reply:
The size varies depending on scanner (vulnerability, misconfiguration and secret) and analyzer (npm and go binary). Adding the flag needs to be carefully designed. We'll merge this PR with the current implementation as this warning is at least useful.

Reply:
Should I go ahead and add a feature enhancement Issue?

Reply:
　Yes, you can open a discussion.